### DISCUSSION TITLE
alert not loading

### DISCUSSION THREAD

**Discussion by XVIIIisthename** [Security-Onion-Solutions/securityonion#11336] [General]
hello i have an error when a try to load the alert section, also the dashboard , the hunt and the cases section. and i have an error in red saying : **_Request failed with status code 502_**
all services a running "ok" and the grid is also on "ok" status
the version is 2.3.260
how can fix this very quick please?

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
Did you install from our Security Onion ISO image or did you perform a network installation?
If network installation, what distro and version did you install on?
How many nodes do you have?
What are the hardware specs of each of those nodes?
How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
Do you get any failures when you run sudo salt-call state.highstate?
Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.
